pkg/linter/linter.go

```diff
@@ -18,153 +18,136 @@ import (
 	"bytes"
 	"context"
 	"debug/elf"
+	"errors"
 	"fmt"
 	"io"
 	"io/fs"
 	"net/http"
 	"os"
 	"path/filepath"
 	"regexp"
+	"slices"
 	"strings"
 
 	"chainguard.dev/apko/pkg/apk/expandapk"
-	linter_defaults "chainguard.dev/melange/pkg/linter/defaults"
+	"github.com/charmbracelet/log"
```

Comment:
I think we're accidentally pulling this in, see my suggestion below about moving the warn vs require distinction outside of this file.